### ISSUE TITLE
PHP Fatal error:  Uncaught Error: Call to undefined function dhcpd_dhcpv4_leasesfile()

### ISSUE THREAD

**Issue by Megachip** [jbaconsult/opnsense_stuff#1]
```
PHP Fatal error:  Uncaught Error: Call to undefined function dhcpd_dhcpv4_leasesfile() in /usr/local/www/widgets/widgets/dhcp_leases.widget.php:56
Stack trace:
#0 /usr/local/www/index.php(409): include()
#1 {main}
  thrown in /usr/local/www/widgets/widgets/dhcp_leases.widget.php on line 56
[06-Jun-2023 16:41:57 Europe/Berlin] PHP Fatal error:  Uncaught Error: Call to undefined function dhcpd_dhcpv4_leasesfile() in /usr/local/www/widgets/widgets/dhcp_leases.widget.php:56
Stack trace:
#0 /usr/local/www/index.php(409): include()
#1 {main}
  thrown in /usr/local/www/widgets/widgets/dhcp_leases.widget.php on line 56
```

**Comment by mk-git** [comment]
See my pull request to fix the error: #2

**Comment by jbaconsult** [comment]
thx. merged!
